### PR TITLE
Feature CI process

### PR DESCRIPTION
**Pull request by n1207n**
This PR addresses Codefresh feature for docker images.

README.md file now has build badges for each container image.

docker-compose uses cleaner image labels.

pylint related dependencies are added.

### CODE REVIEW

File: django_api/requirements/base.txt [hunk 1]
```diff
@@ -8,3 +8,8 @@ djangorestframework==3.6.2
 psycopg2==2.7.1
 django-model-utils==2.6.1  # latest 3.0.0 but need to check changes
 django-mptt==0.8.6
+
+# Testing
+astroid==1.5.2
+isort==4.2.5
+pylint==1.7.1
```
Comment (chachra) [inline django_api/requirements/base.txt]: not used currently correct?

Comment (n1207n) [inline django_api/requirements/base.txt]: Not used in docker container, but it's there if you wanna run pylint locally.

Comment (n1207n) [inline django_api/requirements/base.txt]: This is also reserved for running pylint on codefresh.

**Comment by chachra** [comment]
@n1207n "Nginx proxy: pipeline not found" can deal with it later, but check when you can.

**Pull request merged**